tests/data/dataset_readers/wikitables/world_representation_test.py

```diff
@@ -0,0 +1,20 @@
+# pylint: disable=no-self-use,invalid-name
+from allennlp.data.dataset_readers.wikitables import World
+from allennlp.common.testing import AllenNlpTestCase
+
+
+class TestWorldRepresentation(AllenNlpTestCase):
+    def test_world_processes_sempre_forms_correctly(self):
+        world = World("tests/fixtures/data/wikitables/sample_table.tsv")
+        sempre_form = "((reverse fb:row.row.year) (fb:row.row.league fb:cell.usl_a_league))"
+        expression = world.process_sempre_forms([sempre_form])[0]
+        assert str(expression) == "R(C0,C1(cell:usl_a_league))"
```

Comment:
This looks wrong to me.  Shouldn't it be `R(C0)(C1(cell:usl_a_league)`?

Reply:
NLTK processes it as `R` (reverse) taking two arguments, a function and its argument. This is how it stores it internally.